doc/nrf/device_guides/working_with_nrf/nrf52/developing.rst

```diff
@@ -125,12 +125,16 @@ See how to build the :ref:`peripheral_lbs` sample with MCUboot in direct-xip mod
 Both the :kconfig:option:`CONFIG_MCUBOOT_BOOTLOADER_MODE_DIRECT_XIP` and :kconfig:option:`CONFIG_MCUBOOT_BOOTLOADER_MODE_DIRECT_XIP_WITH_REVERT` Kconfig options automatically enable :kconfig:option:`CONFIG_BOOT_BUILD_DIRECT_XIP_VARIANT`, which allows to build application update images for both slots.
 To see which files are built when the option is enabled, go to the :ref:`app_build_mcuboot_output` page.
 
+.. fota_upgrades_nrfcdm_direct_xip_support_information_start
```

Comment:
`fota_upgrades_over_ble_mcuboot_direct_xip_nrfcdm_note_start`,  `fota_upgrades_over_ble_mcuboot_direct_xip_nrfcdm_note_end` (to improve naming consistency)